distribution/lib/Standard/Base/0.0.0-dev/src/Network/URI.enso

```diff
@@ -72,7 +71,7 @@ type URI
 
              example_scheme = Examples.uri.scheme
     scheme : Text ! Nothing
-    scheme self = Internal.handle_nothing self.internal_uri.getScheme
+    scheme self = handle_nothing self.internal_uri.getScheme
```

Comment:
This may not be for this PR, but IMO this API needs to be revisited.

I highly doubt what we want to do here is to throw `Nothing`. Apart from some control flow, I doubt we want to _throw_ `Nothing` ever - once it is caught somewhere down the dataflow, it's impossible to tell what it originated from or what it means. It's also rather unclear for the user to see 'Nothing' was thrown. If it was a convention we used across all stdlib and users were used to it (it kind of used to be) then maybe it would be ok, but in just a few places, it is a source of confusion IMO.

IMO we should either throw some more meaningful error like `Missing_Field` or something. But ideally, I'd just return `Nothing` as a value instead, making the type not `Text ! Nothing` but `Text | Nothing`.

What do you think @jdunkerley ?

Also, I'm not saying to adapt it in this PR (let's merge it ASAP), but I think may be worth noting so that we revisit it in the near future.